testscommon/state/accountWrapperMock.pb.go

```diff
@@ -26,7 +26,7 @@ var _ = math.Inf
 const _ = proto.GoGoProtoPackageIsVersion3 // please upgrade the proto package
 
 type AccountWrapMockData struct {
-	MockValue int64 `protobuf:"varint,1,opt,name=MockValue,proto3" json:"MockValue,omitempty"`
+	MockValue int64 `protobuf:"varint,1,opt,name=MockValue,proto3" json:"MockValue"`
```

Comment:
This previously had an `omitempty` json tag. However I don't feel this is important. Can remain like this.

Reply:
left it as is, there was no test scenario for omit empty.